modules/rhel-compute-requirements.adoc

```diff
@@ -8,36 +8,39 @@
 [id="rhel-compute-requirements_{context}"]
 = System requirements for RHEL compute nodes
 
-The Red Hat Enterprise Linux (RHEL) compute machine hosts, which are also known as worker machine hosts, in your {product-title} environment must meet the following minimum hardware specifications and system-level requirements.
+The Red Hat Enterprise Linux (RHEL) compute, or worker, machine hosts in your {product-title} environment must meet the following minimum hardware specifications and system-level requirements:
 
 * You must have an active {product-title} subscription on your Red Hat account. If you do not, contact your sales representative for more information.
 
-* Production environments must provide compute machines to support your expected workloads. As a cluster administrator, you must calculate the expected workload and add about 10 percent for overhead. For production environments, allocate enough resources so that a node host failure does not affect your maximum capacity.
+* Production environments must provide compute machines to support your expected workloads. As a cluster administrator, you must calculate the expected workload and add about 10% for overhead. For production environments, allocate enough resources so that a node host failure does not affect your maximum capacity.
 * Each system must meet the following hardware requirements:
 ** Physical or virtual system, or an instance running on a public or private IaaS.
 ifdef::openshift-origin[]
 ** Base OS: CentOS 7.4.
 endif::[]
 ifdef::openshift-enterprise,openshift-webscale[]
-** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/installation_guide/index[RHEL 7.8 - 7.9] with "Minimal" installation option.
+** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/installation_guide/index[RHEL 7.9] or link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/performing_an_advanced_rhel_installation/index[RHEL 8] with "Minimal" installation option.
 +
 [IMPORTANT]
 ====
-Only RHEL 7.8 - 7.9 is supported in {product-title} {product-version}. You must not upgrade your compute machines to RHEL 8.
+Adding RHEL 7.9 compute machines to an {product-title} cluster is deprecated. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+For the most recent list of major functionality that has been deprecated or removed within {product-title}, refer to the _Deprecated and removed features_ section of the {product-title} release notes.
```

Comment:
@katherinedube PTAL about this wording for 7.8+. I will create a separate PR for OCP 4.8 RNs documenting this.

Reply:
LGTM. This deprecation notice seems to be missing in 4.7 as well (even though it's in the 4.6 release notes.)

Reply:
Thanks for pointing out the 4.6 mention, @katherinedube. I didn't realize it was already announced, so I am rethinking the approach here. I've removed the deprecation wording for this PR (because it would only be for 4.8+) and instead, I will open a new PR to apply this to 4.6+.

> Adding RHEL 7 compute machines to an {product-title} cluster is deprecated. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. You must not upgrade your compute machines to RHEL 8.
> 
> For the most recent list of major functionality that has been deprecated or removed within {product-title}, refer to the _Deprecated and removed features_ section of the {product-title} release notes.